workspaces/arborist/lib/arborist/build-ideal-tree.js

```diff
@@ -1527,7 +1528,7 @@ This is a one-time fix-up, please be patient...
 
       const set = optionalSet(node)
       for (const node of set) {
-        node.parent = null
+        node.ideallyInert = true
```

Comment:
I've been thinking about this, and maybe it's better to just call it `inert` instead of `ideallyInert`. I think putting the `ideally` in there is probably a bit too much clever inside baseball and it's more verbose to boot. :)

Some other name would be good, too, as long as it communicates that this is in the tree, but we don't physically install it.